src/sentry/api/serializers/models/integration.py

```diff
@@ -99,10 +100,15 @@ def serialize(self, obj, attrs, user, include_config=True):
             config_data = obj.config
             dynamic_display_information = None
         else:
-            # just doing this to avoid querying for an object we already have
-            installation._org_integration = obj
-            config_data = installation.get_config_data()
-            dynamic_display_information = installation.get_dynamic_display_information()
+            try:
+                # just doing this to avoid querying for an object we already have
+                installation._org_integration = obj
+                config_data = installation.get_config_data()
+                dynamic_display_information = installation.get_dynamic_display_information()
+            except ApiError:
```

Comment:
@NisanthanNanthakumar since we are catching the error here, it means that if something is actually broken on an integrations APIs, we might not catch it because it won't be going to Sentry. We might consider adding Sentry here to capture exceptions. That being said, we know those errors are very noisy so maybe we should just log the errors in an info level log message so we know about them.